README.md

```diff
@@ -3,6 +3,6 @@ I'm Matt, and I make things!
 
 - 🌱 I’m currently looking for game programming internships working in C++, C#, Unity, or Unreal!
 - 🧑‍💻 Learning lower-level APIs such as OpenGL, Vulkan, and more, and trying to build my own engine.
-- 🔭 Studying at Portsmouth University, working on a BSc (Hons), as well as some hobby projects at home!
+- 🔭 Studying a Bachelors in Computer Games Technology at the University of Portsmouth, as well as working on some hobby projects at home!
```

Comment:
careers people were mad that i didn't mention honours, should keep that in

Reply:
used the full name of the course to please them, even linked to the page that says you're approved by tigers or something